src/store/reducers/lots/lotsActions.ts

```diff
@@ -1,9 +1,6 @@
 import { Dispatch } from 'redux';
```

Comment:
please remove `reducers` from the file path

Reply:
done